cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -249,6 +253,57 @@ func (dp *DevicePlugin) getDpdkMounts(dpdkDeviceName string) []pluginapi.Mount {
 	}
 }
 
+func getDeviceCapabilities(device string) string {
+	devID, err := getDeviceID(device)
+	if err != nil {
+		return defaultCapabilities
+	}
+
+	devicesWithCapabilities := map[string]struct{}{
+		"4941": {},
+	}
+
+	if _, ok := devicesWithCapabilities[devID]; !ok {
+		return defaultCapabilities
```

Comment:
Umm, if devicesWithCapabilities is used only here, why do we not just check directly if devID == "4941"?

Reply:
new deviceIDs can be added to that list without changing anything here

Reply:
Ah, I see. I somewhat guessed it may the reason.! Thanks! :)